tools/tests/test_dirinfo.py

```diff
@@ -260,8 +253,7 @@ def test_select_routers(consensus, vote):
 
 
 def test_generate_signed_consensus_from_real_data(consensus, vote):
-    """
-    """
+    """ """
 
```

Comment:
```suggestion
```

Or does that break the test?

Reply:
no, I think someone just forgot to add a comment there...